test/tools/unified-spec-runner/entities.ts

```diff
@@ -337,7 +348,7 @@ export class UnifiedMongoClient extends MongoClient {
   }
 
   get collectedLogs(): LogMessage[] {
-    return this.logCollector.buffer;
+    return (this.options.mongodbLogPath as unknown as { buffer: any[] })?.buffer ?? [];
```

Comment:
Why did you get rid of logCollector?  Seems like it isn't a functional change, and as a result we've lost some type information and need to cast here.